### PR TITLE
Desktopview

### PR DESCRIPTION
**Pull request by mj-e-boc**
-Add index.html, styles.css for desktop view

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,275 +1,389 @@
...
         </p>
-        <h3 class="letsconnect">
-            LET’S CONNECT
-        </h3>
-        <ul class="footericons"><li><a href="#"><img src="./images/Vector-3.svg" alt="githubicon"></a></li>
-        <li><a href="#"><img src="./images/Linkedinicon.svg" alt="linkeinIcon"></a></li>
-        <li><a href="#"><img src="./images/Vector-1.svg" alt="myspaceicon"></a></li>
-        <li><a href="#"><img src="./images/Vector-2.svg" alt="githubicon"></a></li>
-        <li><a href="#"><img src="./images/Vector.svg" alt="evangelisticon"></a></li>
-             </ul>
```
Comment (vickymarz) [inline index.html:42]: - [ ] Good job aligning the content of your headline section the left. 👍🏽 . KIndly make sure that the entire content is at the center of the page to match the figma design.'

Figma design 👇🏽 
image

Your design 👇🏽 
image

File: index.html [hunk 2]
```diff
@@ -1,275 +1,389 @@
...
+          </li>
+          <li>
+            <a href="#"><img src="./images/Vector-2.svg" alt="githubicon" /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./images/Vector.svg" alt="evangelisticon"
+            /></a>
+          </li>
+        </ul>
```
Comment (vickymarz) [inline index.html:66]: - [ ] KIndly reduce the spacing between the social icons to match the figma design 👍🏽 

FIgma design 👇🏽 
image

Your design 👇🏽 
image

File: index.html [hunk 3]
```diff
@@ -1,275 +1,389 @@
...
+      </section>
     </div>
     <!-- Projects section  -->
-<section class="projectscards">
-    <div class="project">
+    <section class="projectscards">
```
Comment (vickymarz) [inline index.html:70]: - [ ] KIndly reduce the spacing between your headline section and the projects section to match the figma design 👍🏽 

Figma design 👇🏽 
image

Your design 👇🏽 
image

File: index.html [hunk 4]
```diff
@@ -1,275 +1,389 @@
...
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
```
Comment (vickymarz) [inline index.html:105]: - [ ] KIndly reduce the font size of all your list of technologies to match the figma design. KIndly check the figma design property values 💯 

FIgma design 👇🏽 
image

Your design 👇🏽 
image

File: index.html [hunk 5]
```diff
@@ -1,275 +1,389 @@
...
+          <button type="button" class="button">See Project</button>
```
Comment (vickymarz) [inline index.html:106]: - [ ] KIndly use the appropriate background color for the button as it is on the figma design  👍🏽 

YOur design 👇🏽 
image

FIgma design 👇🏽 
image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @mj-e-boc , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅  
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] KIndly reduce the font size of all your list of technologies to match the figma design. KIndly check the figma design property values 💯 

FIgma design 👇🏽 
image

Your design 👇🏽 
image

- [ ] Kindly use the appropriate background color for the projects card and about me button as it is on the figma design  👍🏽 

Your design 👇🏽 
image

FIgma design 👇🏽 
image



_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @mj-e-boc , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image


 ## Highlights
 - Great work using the right GitHub Flow 💯 
 - Linters are passing 👍🏾 
 - Project Documentation (PR, README) is professional 👏🏾 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 6]
```diff
@@ -375,3 +386,183 @@ textarea {
...
+  }
+
+  .getresumebutton {
+    width: 20%;
+    padding: 2%;
+  }
+
+  .abtme {
+    display: flex;
+  }
```
Comment (AlphaNtihinduka) [inline styles.css:462]: - [ ] [OPTIONAL] I have noticed that your `.abtme` has a `padding: 20% 5%`. This causes high spacing which does not stick to the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=NUG5R401LrO7ZCOa-0) :no_good: Please, consider reducing the padding top and bottom of the section. 
- Figma design :sunglasses: 
Screenshot from 2023-01-20 04-34-32

- Your design :sunglasses: 
Screenshot from 2023-01-20 04-35-55

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @mj-e-boc     🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

File: styles.css [hunk 7]
```diff
@@ -375,3 +375,82 @@ textarea {
...
+
+/* desktopview */
+
+@media(min-width:768px){
+
+.abtme{
+  display: flex;
+}
+
+.textitems{}
```
Comment (alexiscyber14) [inline styles.css]: Please try to not keep empty blocks of Code. This will help your code be more organized.

File: styles.css [hunk 8]
```diff
@@ -417,8 +417,8 @@ textarea {
   }
 
   .project .button {
-    width: 30%;
-    padding: 4%;
+    width: 25%;
+    padding: 2%;
   }
 
   .pro2 button {
```
Comment (alexiscyber14) [inline styles.css:424]: Please try and remove this empty block of code as well. from line 327-417

**Pull request merged**